lua/conform/formatters/python-ly.lua

```diff
@@ -0,0 +1,9 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/frescobaldi/python-ly",
+    description = "A Python package and commandline tool to manipulate LilyPond files ",
+  },
+  command = "ly",
+  args = { "reformat", "$FILENAME" },
```

Comment:
You're passing in the filename, but the default value for `stdin` is true. Does this formatter work via stdin/stdout or does it operate on a file?

Reply:
My mistake, now it is correct